ansible/playbook.yml

```diff
@@ -150,6 +153,23 @@
       shell:
         cmd: "for fl in /usr/lib/postgresql/bin/* ; do ln -sf $fl /usr/bin/$(basename $fl) ; done"
 
+    - name: Enable pgTAP extension
```

Comment:
let's move these out to a separate task file that gets included

Reply:
okay, will update the PR